ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestSuite.scala

```diff
@@ -126,4 +126,41 @@ private[testtool] abstract class LedgerTestSuite(val session: LedgerSession) {
     }
   }
 
+  implicit def diffShowSeq[T](implicit diffShowT: DiffShow[T]): DiffShow[Seq[T]] =
+    new DiffShow[Seq[T]] {
+      override def show(t: Seq[T]): String = t.toString
+
+      override def diff(left: Seq[T], right: Seq[T]): Comparison = {
+        val leftList = left.toList
+        val rightList = right.toList
+        val changed = leftList
+          .zip(rightList)
+          .zipWithIndex
+          .map { case ((l, r), index) => index -> diffShowT.diff(l, r) }
+          .filter { case (_, diff) => !diff.isIdentical }
+          .map { case (index, diff) => index.toString -> diff.string }
```

Comment:
Minor: a `filter` followed by a `map` is usually a good use case for a `collect`.
```suggestion
          .collect { case (index, diff) if !diff.isIdentical => index.toString -> diff.string }
```

Reply:
Agreed, I was flipping back and forth for readability but I think this is fine.